homeassistant/components/hue.py

```diff
@@ -201,6 +201,9 @@ def setup(self):
                             self.host)
             self.request_configuration()
             return
+        except Exception:  # pylint: disable=broad-except
+            _LOGGER.exception("Unknown error connecting with Hue bridge at %s",
+                              self.host)
 
```

Comment:
We should probably return here.

Reply:
Good point. Will include it in my #12830 branch